### PR TITLE
highlight pipe

### PR DESCRIPTION
**Pull request by munkk**

### CODE REVIEW

File: src/pipes/highlight/highlight.pipe.test.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { HighlightPipe } from './highlight.pipe';
+
+describe('Pipe: HighlightPipe', () => {
+	let pipe: HighlightPipe;
+
+	beforeEach(() => {
+		pipe = new HighlightPipe();
+	});
+
+	it('should wrap specified string with `q-grid-highlight-part` class', () => {
```
Comment (vkorolev) [inline src/pipes/highlight/highlight.pipe.test.ts]: Consider to add tests for edge cases: when text/search is null/undefined/0,

File: src/pipes/highlight/highlight.pipe.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { GRID_PREFIX } from '@grid/view/definition';
```
Comment (klumba12) [inline src/pipes/highlight/highlight.pipe.ts]: does it work?

File: src/pipes/highlight/highlight.pipe.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { GRID_PREFIX } from 'ng2-qgrid/core/definition';
+
+@Pipe({
+	name: 'highlight'
```
Comment (klumba12) [inline src/pipes/highlight/highlight.pipe.ts]: rename to qGridHighlight

File: src/pipes/highlight/highlight.pipe.ts [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+			if (contained.test(search)) {
+				return `<span class="${GRID_PREFIX}-highlight-part">${htmlEncode(text)}</span>`;
+			}
+		}
+
+		return htmlEncode(text);
+	}
+}
+
+function htmlEncode(s: string): string {
```
Comment (klumba12) [inline src/pipes/highlight/highlight.pipe.ts]: can you add htmlEncode escapeRegexp under core/utility.js?
